### DISCUSSION TITLE
Crash on opening (2022.108.0)

### DISCUSSION THREAD

**Discussion by Syrade** [ppy/osu#16365] [Q&A]
So, I downloaded the Osu.App.Zip from the releases.  I open the application, it crashes and I'm told repeatedly that it crashed unexpectedly.
![Screen Shot 2022-01-08 at 10 18 45 AM](https://user-images.githubusercontent.com/89738145/148649613-a0e273d8-5931-4e2f-9ef7-28530ef1d1c1.png)
.

These are my computer specs.
![Screen Shot 2022-01-08 at 10 22 14 AM](https://user-images.githubusercontent.com/89738145/148650654-fb09d93f-4ebf-4781-9380-84908cfdfa54.png)

I don't think I can link logs because i wasn't able to open the game, it just instantly crashed. I'm using Mac 10.12.6 on Mac Sierra.

**Comment by bdach** [comment]
[please provide logs to help us troubleshoot your problems](https://github.com/ppy/osu/discussions/15188)

**Reply by Syrade** [reply]
It's not sending me to a logs folder.

**Comment by peppy** [comment]
Hard to tell if dotnet is supported on said macOS release, it's pretty ancient... (I can't attain whether or not from the official page: https://docs.microsoft.com/en-us/dotnet/core/install/macos#unsupported-releases)

**Reply by Syrade** [reply]
Looks like 10.12 isn't supported for dotnet, is there an earlier release for this type of problem?

**Reply by peppy** [reply]
Very hard for us to maintain backward compatibility support.

You can go back in the [release history](https://github.com/ppy/osu/releases) and try and find a working version, probably before we switched to .NET5. But it'll be a bit ancient.

**Reply by Syrade** [reply]
Do you remember when you started using .NET5?  I just need the year, not version number.

**Reply by Syrade** [reply]
Nevermind, I searched up when .NET5 was made.

**Reply by bdach** [reply]
last release not on .net 5 would be https://osu.ppy.sh/home/changelog/lazer/2021.109.0

**Reply by peppy** [reply]
If 10.12 is no longer supported we may need to update our readme, still says "10.12+".

Need to confirm this is the case first, though.
